chipsec/helper/oshelper.py

```diff
@@ -100,13 +99,13 @@ class OsHelper:
     def __init__(self):
         self.helper = None
         self.loadHelpers()
-        #print "Operating System: %s %s %s %s" % (self.os_system, self.os_release, self.os_version, self.os_machine)
+        #print "Operating System: {} {} {} {}".format(self.os_system, self.os_release, self.os_version, self.os_machine)
```

Comment:
Why are we fixing dead code.  Please remove the dead code.  It happens in a number of locations in this file.

Reply:
ok will remove